draft-ietf-teep-architecture.md

```diff
@@ -1329,23 +1329,42 @@ may need to be updated.
-operator. To mitigate this threat, TEEs have several options for 
-detecting and mitigating a compromised TAM:
-
-1. Apply an ACL to the TAM, indicating which TAs it is permitted to install.
-2. Use a transparency log to expose a TA compromise: TAMs publish an 
-out-of-band record of TA releases, allowing a TEE to cross-check the TAs
-delivered against the TA releases in order to detect a TAM compromise.
+to determine that the TAM is trustworthy. A compromised TAM may bring multiple threats 
+and damage to user devices that it can manage and thus to the Device Owners. 
+Information on devices that the TAM manages may be leaked to a bad actor.
+A compromised TAM can also install many TAs to launch a DoS attack on devices,
+for example, by filling up a device's TEE resources reserved for TAs such that 
+other TAs may not get resources to be installed or properly function. It may
+also install malicious TAs to potentially many devices under the condition that 
+it also has a Trusted Component signer key that is trusted by the TEEs.
+This makes TAMs high value targets. A TAM could be compromised without impacting its certificate or 
+raising concern from the TAM's operator.
+
+To mitigate this threat, TEEP Agents and Device Owners have several options, 
+including but potentially not limited to those listed below, for detecting and mitigating a compromised TAM:
+
+1. Apply an ACL to the TAM key, indicating which TAs the TAM is permitted to install.
+2. Use a transparency log to expose a TAM compromise: TAMs publish an 
+out-of-band record of Trusted Component releases, allowing a TEE to cross-check the Trusted Components
+delivered against the Trusted Component installs in order to detect a TAM compromise.
 3. Use remote attestation of the TAM to prove trustworthiness.
 
-While TAs are signed by the creator and delivered by the TAM, there is
-still a threat present in allowing a potentially compromised TAM to install
-any TA it wants; for example, a known-compromised TA or an explicitly
-malicious TA.
+## Hostile TAM
+
+There are also threats of hostile or abusive TAMs that act out of expectation 
+of Device Administrators, for example, pushing out TAs that contain 
+some data collection or use device resources for distributed jobs for 
+a TAM. All the potential damages that a compromised TAM can make can be done by a
+hostile TAM. A hostile TAM differs in intentionally conducting some abusive behavior.
+
+A Device Administrator may find out that the TAM it selected starts to become hostile
+or abusive. In this case, the Device Administrator can remove
+those TAs managed by that TAM from its devices, and stop trusting the TAM by
+updating the TEEP Agent's trust anchor store to use another TAM that can deliver legitimate
+TAs it needs.
+
+A new hostile TAM may be introduced if the Trust Anchor store in a TEEP Agent is compromised.
+See Section {{#trust-anchor-compromise}} above for Trust Anchor protection.
```

Comment:
As noted in email, I would prefer not adding this section.

Reply:
Will remove it, and just leave the expanded Compromised TAM to cover both cases. We will respond Ben for this reasoning.